### PR TITLE
POLIO-1082 wrong vaccine scope

### PR DESCRIPTION
**Pull request by quang-le**
The color code in the calendar was wrong if users had input a scope for a vaccine then deleted it

Related JIRA tickets :POLIO-1082

## Self proofreading checklist

- [X] Did I use eslint and black formatters
- [X] Is my code clear enough and well documented
- [ ] Are my typescript files well typed
- [ ] New translations have been added or updated if new strings have been introduced in the frontend
- [ ] My migrations file are included
- [ ] Are there enough tests
- [ ] Documentation has been included (for new feature)

## Changes

- add "org_units"  to the `Group` fields returned by the `CalendarCampaignSerializer`
- filter out scopes without org units in the front end

## How to test

Explain how to test your PR.

Go to calendar, edit a campaign to add, then remove a vaccine scope: the calendar should update accordingly

## Print screen / video

https://github.com/BLSQ/iaso/assets/38907762/7fb867da-96e5-43a3-aaeb-6a35faec2b4e

## Notes
depends on https://github.com/BLSQ/iaso/pull/683

### CODE REVIEW

File: plugins/polio/serializers.py [hunk 1]
```diff
@@ -874,7 +874,7 @@ class NestedScopeSerializer(RoundScopeSerializer):
             class NestedGroupSerializer(GroupSerializer):
                 class Meta:
                     model = Group
-                    fields = ["id"]
+                    fields = ["id", "org_units"]
```
Comment (madewulf) [inline plugins/polio/serializers.py]: Isn't this too costly?  Wouldn't this end up serializing all districts for the AFRO zone on each calendar refresh?

Comment (madewulf) [reply plugins/polio/serializers.py]: This is a lot of integer to send through the api while there is no need for all the details 
CleanShot 2023-07-05 at 22 06 10@2x

Comment (quang-le) [reply plugins/polio/serializers.py]: Updated to use values from backend. the PR now depends on POLIO-846

File: plugins/polio/js/src/components/campaignCalendar/cells/RoundCell.js [hunk 2]
```diff
@@ -40,21 +44,22 @@ const RoundCell = ({ colSpan, campaign, round }) => {
...
+            ? round.vaccine_names?.split(',') ?? []
+            : campaign.original.vaccines?.split(',') ?? [];
+        return list.map(vaccineName => vaccineName.trim());
+    }, [
+        campaign.original.vaccines,
+        campaign.separateScopesPerRound,
+        round.vaccine_names,
+    ]);
+    // const getVaccineColor = useCallback(
+    //     vaccine =>
```
Comment (hakifran) [inline plugins/polio/js/src/components/campaignCalendar/cells/RoundCell.js]: Is there any reason to keep this code in comment?

**Review by hakifran — APPROVED** [review]
Seems good for me. But there are conflict

**Review by madewulf — APPROVED** [review]

**Review by beygorghor — APPROVED** [review]

**Pull request merged**